lib/datastore/request.js

```diff
@@ -148,6 +148,15 @@ DatastoreRequest.prototype.get = function(keys, callback) {
 };
 
 /**
+ * Maps to {module:datastore/dataset#save}, forcing the method to be `insert`.
+ */
+DatastoreRequest.prototype.insert = function(entities, callback) {
+  var isArray = util.is(entities, 'array');
+  entities = util.arrayize(entities).map(util.propAssign('method', 'insert'));
+  this.save(isArray ? entities : entities[0], callback);
```

Comment:
I see that being used in `get` and `delete` but not in `save`? Could you point me to the logic?